envs/.env.example

```diff
@@ -35,6 +35,11 @@ NEXT_PUBLIC_GOOGLE_TAG_MANAGER_ID=GTM-WFJWBD
 NEXT_PUBLIC_GOOGLE_TAG_MANAGER_AUTH=N9BisSDKAwJENFQtQIEvXQ
 NEXT_PUBLIC_GOOGLE_TAG_MANAGER_PREVIEW=env-423
 
+# Datadog Connection
+NEXT_PUBLIC_DATADOG_RUM_APP_ID=Retrieve this from Datadog RUM settings - Search 'Nextjs' for the application
+NEXT_PUBLIC_DATADOG_RUM_CLIENT_TOKEN=Retrieve this from Datadog RUM settings - Search 'Nextjs' for the application
+NEXT_PUBLIC_DATADOG_RUM_SERVICE=Retrieve this from Datadog RUM settings - Search 'Nextjs' for the application
```

Comment:
I would probably leave these blank and put the instructions in comments; otherwise the env var has a value and that might be confusing.